contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -370,11 +370,13 @@ contract LoanTokenLogicStandard is LoanTokenSettingsLowerAdmin {
 			sentAmounts[1] /// depositAmount
 		);
 
+		/// @dev Converting to initialMargin
+		leverageAmount = SafeMath.div(10**38, leverageAmount);
```

Comment:
this like doesn't look great - you should declare a new variable instead of reusing existing one.
I guess you are trying to avoid "Stack too deep" issue here, consider inlining this logic into return statement below

Reply:
tried that too. also gave a stack too deep error.

Reply:
btw: only commented out the tests for the removed functions - which could be reintroduced with the refactored version (code size issues).